apps/full-site-editing/full-site-editing-plugin/blocks/content-slot/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import './style.scss';
+
+registerBlockType( 'a8c/content-slot', {
+	title: __( 'Content Slot' ),
+	description: __( 'Placeholder for a post or a page.' ),
+	icon: 'layout',
+	category: 'layout',
+	supports: {
+		align: [ 'wide', 'full' ],
+		anchor: true,
+		html: false,
+		multiple: false,
+		reusable: false,
+	},
+	edit,
+	save: () => null,
```

Comment:
at a later point in time or even now I think it's worth spitting some output here.
this is somewhat high-level and not a priority at this point, but one of our guiding principles is being part of the open web. I would love to see some fallback message here for the case where a post is viewed purely as HTML without WordPress the PHP engine rendering it.

could we think of a simple message to report that would convey "this is the spot where a post would normally show, but this page is being viewed without the necessary software so you get this instead 😄 "